### ISSUE TITLE
Regression: Alt text missing from title tag in media since 4.4

### ISSUE THREAD

**Issue by gsuberland** [mastodon/mastodon#35466]
### Steps to reproduce the problem

1. Post an image or video with alt text
2. See it in your feed or notifications tab
3. Hover your mouse over it

### Expected behaviour

Alt text should appear as it did before the 4.4 release.

### Actual behaviour

Alt text does not appear because the title attribute is not set.

### Detailed description

The title attribute is set on images and videos in the expanded media card, but is not set on images and videos inline in feeds. This is a regression in the 4.4 release.

I usually read the alt text on all images and videos simply by hovering my mouse over them in-feed, but I can no longer do that since the update and have to expand the post to read it.

### Mastodon instance

chaos.social

### Mastodon version

v4.4.1

### Browser name and version

Firefox 140.0.4

### Operating system

Windows 10

### Technical details

This additionally breaks a common CSS addon that highlights images and videos that do not have alt text. Chaos Social has this CSS feature and it has been broken since 4.4 due to this regression. Details here: https://chaos.social/@gsuberland/114896076256164082

**Comment by Gargron** [comment]
Your CSS add-on should be checking for the `alt` tag, not `title`.

**Comment by gsuberland** [comment]
Yeah, I'll poke the chaos admins to get that one fixed. Sorry I missed the existing issue; I did search before creating a new issue but for some reason that one didn't pop up.
